compiler/src/main/java/com/squareup/anvil/compiler/codegen/ContributesSubcomponentHandlerGenerator.kt

```diff
@@ -350,41 +349,40 @@ internal class ContributesSubcomponentHandlerGenerator(
     factoryClass: FqName?
   ): ParentComponentInterfaceHolder? {
     val contributionFqName = contribution.clazzFqName
-    val contributionDescriptor = contributionFqName.requireClassDescriptor(module)
 
-    val contributedInnerComponentInterfaces = contributionDescriptor.unsubstitutedMemberScope
-      .getContributedDescriptors(kindFilter = CLASSIFIERS)
-      .filterIsInstance<ClassDescriptor>()
-      .filter { DescriptorUtils.isInterface(it) }
-      .filter {
-        val annotation = it.annotationOrNull(contributesToFqName, contribution.parentScope)
-        annotation != null
+    val contributedInnerComponentInterfaces = contribution.classReference
```

Comment:
Note that we used descriptor APIs before, and this was failing when a subcomponent was generated through a code generator. With the abstract API this isn't an issue anymore.